site/quickstart.md

```diff
@@ -0,0 +1,70 @@
+
+## 3. Configure Kibana and Elasticsearch.
+Invoke the script to create these. (Again, list the commands here using an example.)
+
+## 4. Install the operator.
+* Create a namespace and service account for the operator (point to sample script that does this.)
+* Invoke the script to generate the credentials for the operator and add it to the the operator YAML file (all the values can be left as default).
+* Create the operator using `helm install`, and passing in the namespace, service account, and location of Elasticsearch (Point to Helm.)
+
+## 5. Prepare your environment for a domain.
+* Create a domain namespace
+* Create the Kubernetes secrets for the Administration Server boot credentials
+* Create a PV & PVC domain; use script
+* Configure the operator to manage the domains in the domain namespace using `helm upgrade`
+* Configure the Traefik load balancer to manage the domains in the domain namespace.
+  * Use `helm upgrade` to configure the load balancer to monitor the Ingresses in the domain namespace
+* Create the domain home
+* Create the Docker image for the domain home in the image or use the WebLogic binary image.
+  * Run WLST to create the domain in PV (remember to apply patch).
+
+## 6. Create a domain.
+* Edit the domain YAML file (can the defaults be used?)
+* Create the domain home for the domain
+  * For a domain home on PV – as in sample
+  * For a domain home in image, use the sample in the Docker GitHub project
+* Optionally, create a configuration overrides template and any additional Kubernetes secrets it needs (for example, to override the domain home configuration of a database URL, username, and password).
+* Create a domain resource in the domain namespace
+  * Specify the following information: domain UID, service account, secret name, the domain home details, and optionally, the configuration overrides template name.
+* Configure the operator to know about the domain
+* Configure the Traefik load balancer to manage the domain as follows:
+  * Create an Ingress for the domain in the domain namespace (it contains the routing rules for the domain)
+
+(At this point, do they have a WebLogic Kubernetes deployment in a Kubernetes cluster? If so, we have to give them something to look at, to verify their results.)
+
+(Removing the domain, operator, and such, isn't really needed in a quick start guide. We can point them to the user's instructions for how to remove the domain, operator, and all the other related resources.)
+
+## 7. Remove a domain.
+* Remove the domain's Kubernetes resources (domain, secrets, ingress, ...)
+  * If they are all tagged with a `weblogicUID` label, then the sample delete domain script will remove them all  
+  * The operator will notice that the domain's domain resource has been removed and will then kill the pods
+* Configure the Traefik load balancer to stop managing the domain
```

Comment:
If you have configured Traefik to manage the domain's namespace (instead of default: all namespaces) then edit Traefik's yaml to remove domain1 namespace and do a 'helm update'
helm update --name traefik-operator --namespace traefik (default values in yaml)
or
helm update --name traefik-operator --namespace traefik --values values.yaml stable/traefik

Reply:
Got it.